src/openrct2/ride/gentle/Maze.cpp

```diff
@@ -181,3 +201,132 @@ TRACK_PAINT_FUNCTION get_track_paint_function_maze(int32_t trackType, int32_t di
 
     return maze_paint_setup;
 }
+
+#pragma region Maze Pathfinding
+
+void swap(MazePathfindingEntry& a, MazePathfindingEntry& b)
```

Comment:
I don't think this function should exist

Reply:
Fixed by Fixed by 1faeb39.